diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -34,17 +34,30 @@ import java.io.File
  */
 const val DIKTAT_RULE_SET_ID = "diktat-ruleset"
 
-class DiktatRuleSetProvider(private val diktatConfigFile: String = "diktat-analysis.yml") : RuleSetProvider {
+// by the default it is expected to have diktat-analysis.yml configuration in the root folder where ktlint is run
+// otherwise it will use default configuration where some rules are disabled
```

Comment:
```suggestion
/**
 * by the default it is expected to have diktat-analysis.yml configuration in the root folder where ktlint is run
 * otherwise it will use default configuration where some rules are disabled
 */ 
```

Reply:
fixed in some other way